compute/sqlserver/powershell/create-sql-instance-availability-group-prerequisites.ps1

```diff
@@ -0,0 +1,384 @@
+﻿#Requires -Version 5
+# Copyright(c) 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+##############################################################################
+#.SYNOPSIS
+# Run the prerequisites before running the script: 
+#   create-sql-instance-availability-group.ps1
+#
+#.DESCRIPTION
+# The script create-sql-instance-availability-group.ps1 assumes you have 
+# custom network with 3 subnetworks. It also expects a Windows AD domain.
+# 
+# This script can be used to create all of those prerequisites
+#
+# You should run this script one section at a time since not all sections
+# may apply in your case
+#
+#.NOTES
+# AUTHOR: Anibal Santiago - @SQLThinker
+##############################################################################
+
+
+################################################################################
+# Specify the parameters
+################################################################################
+# Network configuration
+$network      = 'wsfcnet'      # Name of the Custom network

```

Comment:
It's not clear if users are expected to update these parameters with their own settings.